packer/template-ubuntu-14.04.json

```diff
@@ -131,9 +145,46 @@
         "ssh_wait_timeout": "10000s",
         "shutdown_command": "echo 'shutdown -P now' > /tmp/shutdown.sh; echo 'vagrant'|sudo -S sh '/tmp/shutdown.sh'",
         "vmx_data": {
-          "memsize": "512",
-          "numvcpus": "1",
-          "cpuid.coresPerSocket": "1"
+            "memsize": "1024",
+            "numvcpus": "2",
+            "cpuid.coresPerSocket": "1",
+
+            "ethernet0.virtualDev": "e1000",
+            "ethernet0.present": "TRUE",
+            "ethernet0.connectiontype": "nat",
+            "ethernet0.pcislotnumber": "32",
+
+            "ethernet1.virtualDev": "e1000",
+            "ethernet1.present": "TRUE",
+            "ethernet1.pcislotnumber": "33"
+
+
+        },
+        "vmx_data_post":
+        {
+            "memsize": "1024",
+            "numvcpus": "2",
+            "cpuid.coresPerSocket": "1",
+
+
+            "ethernet0.networkName": "ADMIN",
```

Comment:
Sounds like that to me at first.
 but in OnRack convention, ADMIN is a network channel to "administrate" the ORA(RackHD) , BMC channel to a separated channel to talk with BMCs( in some Lab, BMCs are in another network )